app/scripts/controllers/detect-tokens.js

```diff
@@ -116,7 +116,7 @@ export default class DetectTokensController {
     const tokensToDetect = [];
     for (const tokenAddress in tokenListUsed) {
       if (
-        !this.tokenAddresses.find(({ address }) =>
+        !this.tokenAddresses.find((address) =>
```

Comment:
why was this change required?

Reply:
This change is correct, seems that previous code was wrong.

Reply:
lgtm

I think we'll want to update line 122 as well.  https://github.com/MetaMask/metamask-extension/pull/16772/files#r1055712836